package.json

```diff
@@ -76,6 +76,8 @@
 		"build-static": "npx ncp static public",
 		"prebuild-server": "mkdirp build",
 		"build-server": "BROWSERSLIST_ENV=server webpack --config client/webpack.config.node.js --display errors-only",
+		"build-server:copy-modules": "node bin/copy-production-modules.js",
+		"postbuild-server": "node -e \"process.env.CALYPSO_ENV === 'production' && process.exit(1)\" || yarn run build-server:copy-modules",
```

Comment:
Should we do this in other envs as well like `stage` or `wpcalypso`?

Reply:
The `CALYPSO_ENV` variable is a bit weird as it has two meanings.

During build, the values used in practice are `development`, `production` and `desktop`. There's also `desktop-development`, but I think that setup is not functional at this moment.

We never use `stage` or `wpcalypso` at build time. These values are used only in the second scenario: setting an environment variable at runtime. At runtime, we always run the same build produced by the `production` setup, and it's instructed to load the `stage` or `wpcalypso` configs by the `CALYPSO_ENV` environment variable.